docs/programming-languages/javascript-and-node-js.md

```diff
@@ -173,7 +173,7 @@ for details on pre-installed browsers and testing tools on Semaphore.
 [tutorial]: https://docs.semaphoreci.com/examples/node-js-and-typescript-continuous-integration/
 [demo-project]: https://github.com/semaphoreci-demos/semaphore-demo-javascript
 [ubuntu-javascript]: https://docs.semaphoreci.com/ci-cd-environment/ubuntu-18.04-image/#javascript-via-node-js
-[macos-javascript]: https://docs.semaphoreci.com/ci-cd-environment/macos-mojave-xcode-11-image/#javascript-via-node-js
+[macos-javascript]: https://docs.semaphoreci.com/ci-cd-environment/macos-xcode-11-image/#javascript-via-node-js
```

Comment:
This link is a bit different, it should be, there is no dash between node and js on the page so it should be: https://docs.semaphoreci.com/ci-cd-environment/macos-xcode-11-image/#javascript-via-nodejs